umh-core/pkg/constants/redpanda_monitor.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2025 UMH Systems GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package constants
+
+import "time"
+
+// RedpandaMonitorUpdateObservedStateTimeout is the timeout for updating the observed state
+const RedpandaMonitorUpdateObservedStateTimeout = 2 * time.Millisecond
```

Comment:
double check with these changes: https://github.com/united-manufacturing-hub/united-manufacturing-hub/pull/1982/files#diff-f36e8417aa9923ee4d384078c99f84e5c5e68f922dba97b50331810ceb0a710bR29-R30

Reply:
(I added there a seperate processor timeout for redpanda montior)

see also these changes: https://github.com/united-manufacturing-hub/united-manufacturing-hub/pull/1982/files#diff-9f3c57d4c24c62ee440ffa411f43f5c1b460c4acc3706b4954a8a108f69af828L390-R432

Reply:
These are also applied.